Source/Scene/GroundPrimitive.js

```diff
@@ -111,7 +115,7 @@ define([
      * scene.primitives.add(new Cesium.GroundPrimitive({
      *   geometryInstance : rectangleInstance
```

Comment:
Update the sample to not use the deprecated parameter.  Perhaps also add a second example showing batch with a comment saying that the geometries are the same color.